community/sustainability_effort.md

```diff
@@ -0,0 +1,61 @@
+# Argo Workflows Sustainability Effort
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
+
+We are trying an Incentive System in which contributors pledge a certain number of hours per week (average) to the project and in return have their PRs prioritized for review.
+
+## Where is help needed?
+
+Help is needed for:
```

Comment:
Gosh, I'm not sure if this is necessarily the priority order, though. In fact, I think the priority order for an individual is based on how they can best help. And below it says the highest priority is to address the PRs with the "prioritized" label, as opposed to PRs in general. It may be sufficient to make that the only priority, and everything else is just considered valuable.?

Reply:
Oh. Well that's true I suppose (although usually implied). I suppose the priority order makes more sense specifically for folks who are in fact capable of doing all of the above, which are indeed generally Reviewer+

If we want to incentivize reviewing PRs in general (as the area with the most need), I do think PRs should still be the highest priority if possible, and `prioritized` PRs above that.

But yea I think we can leave this as is then and only mention that below for Reviewer+. Let's make that a sub-bullet as opposed to parenthesized in the below section then

Reply:
will do once I'm on a device where I can